frontend/src/views/main/project/TestSuiteExecutions.vue

```diff
@@ -154,6 +154,11 @@ const executionBreadcrumbs = computed(() =>
       }
     ]);
 
+function executionResults(execution: TestSuiteExecutionDTO): boolean[] {
+  return execution.results ? execution.results.map(result => result.passed) : [];
+}
+
+useRouterParamSynchronization('test-suite-new-execution', 'executionId', executions, selectedExecution, 'id');
```

Comment:
I think this can be also replaced by standard vue features: you can add a component input of `executionDate` (the name is also misleading since it's an id, not date) with `defineProps` and react to it's change to get the whole `selectedExecution` object by id from the store